lib/new_relic/recipes/capistrano3.rb

```diff
@@ -9,46 +9,56 @@
   # notifies New Relic of a deployment
   desc "Record a deployment in New Relic (newrelic.com)"
   task :notice_deployment do
-    run_locally do
-      environment = fetch(:newrelic_rails_env, fetch(:rack_env, fetch(:rails_env, fetch(:stage, "production"))))
+    if fetch(:newrelic_role)
+      on roles(fetch(:newrelic_role)) do
+        notice_deploymet
+      end
+    else
+      run_locally do
+        notice_deploymet
+      end
+    end
+  end
 
-      require 'new_relic/cli/command.rb'
+  def notice_deploymet
```

Comment:
Hi @NielsKSchjoedt thanks for the contribution, it does seem like this will be useful! We were thinking that perhaps it might be a bit confusing to see a method name here that's so similar to the task name. Perhaps something like `upload_deployment` or `send_deployment` or some other variation? Open to your ideas.